addons/mail/static/src/components/thread_preview/thread_preview.js

```diff
@@ -68,7 +68,7 @@ class ThreadPreview extends Component {
         if (!this.thread.lastMessage) {
             return '';
         }
-        return mailUtils.parseAndTransform(this.thread.lastMessage.prettyBody, mailUtils.inline);
+        return mailUtils.htmlToTextContentInline(this.thread.lastMessage.prettyBody);
```

Comment:
I would add a comment to `parseAndTransform` method to indicate that it is unsafe and should never be used for HTML that we intend to inject with t-raw to avoid any potential future issue (and to advise to use htmlToTextContentInline instead), and I would double check all current uses of that method.

Reply:
But I'm surprised checking the code.

Are you sure the injection was possible? Wasn't it just that script was totally removed whereas the other method escapes it ?

Reply:
The old test I took back, after adaptation to the new implementation was failing because of an injection so I suppose yeah. But in fact as I said, `body` is sanitized in python so normally it's not a big deal unless we store the message body directly from js creation (ex if we post a message and we don't take the result from postMessage rpc to create the message but directly the text from composer)

Reply:
Ok well then if parseAndTransform is indeed not safe, we should add the comment I suggested above

Reply:
It's not that it's not safe, it depends on the transformFunction used, I tried to write a comment to explain it clearly, tell me if you think it's clear enough :)